build.sbt

```diff
@@ -44,3 +61,7 @@ lazy val testing = (project in file("testing"))
       Libraries.discipline
     )
   )
+
+lazy val `circe-yaml` = (project in file (".")).settings(
+  commonSettings
+) dependsOn (snake) //aggregate (snake)
```

Comment:
OK, for 0.3.1-SNAPSHOT we can change it back to one flat base module. I don't think it really matters that much though, and this way does make it easier to add submodules if they become necessary. The root project depending on the snakeYAML project should make it the same thing from the user's perspective anyway.